src/main/java/com/dougnoel/sentinel/elements/WindowsElement.java

```diff
@@ -244,9 +228,62 @@ public Color getColor() {
 	@Override
 	public boolean attributeEquals(String attribute, String value) {
 		if (attribute.equalsIgnoreCase("color")) {
-			var hexColor = getColor().asHex();
+			var hexColor = getColorAtOffset().asHex();
 			return hexColor.equalsIgnoreCase(value);
 		}
 		return super.attributeEquals(attribute, value);
 	}
+
+	/**
+	 * Windows does not support color information, as such this is an unsupported operation for windows elements.
+	 *
+	 * @return NotImplementedException
+	 */
+	@Override
+	public java.awt.Color getBackgroundColor() throws NotImplementedException {
+		throw new NotImplementedException("Windows elements do not support background color retrieving");
+	}
+
+	/**
+	 * Returns a screenshot File of the current element.
+	 *
+	 * @return File a screenshot of the current element
+	 */
+	@Override
+	public File getScreenshot() {
+		BufferedImage temporaryBuffer = screenshotElement();
+		try{
+			return FileManager.saveImage(null, "tempScreenshotImage.png", temporaryBuffer);
+		}
+		catch(IOException e){
```

Comment:
Spacing inconsistent on this try{}catch(){

Reply:
Kinda confused what you mean.